pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -188,10 +188,17 @@ let
         moveToOutput "bin/pg_config" "$dev"
         # To prevent a "pg_config: could not find own program executable" error, we fake
         # pg_config in the default output.
-        cat << EOF > "$out/bin/pg_config" && chmod +x "$out/bin/pg_config"
+        cat << 'EOF' > "$out/bin/pg_config" && chmod +x "$out/bin/pg_config"
         #!${stdenv'.shell}
-        echo The real pg_config can be found in the -dev output.
-        exit 1
+        my_path="$(readlink -f -- "$0")"
+        on_path="$(readlink -f -- "$(which pg_config)")"
```

Comment:
I wouldn't count on `which` being available, `type` may be a better option here as it's a builtin?